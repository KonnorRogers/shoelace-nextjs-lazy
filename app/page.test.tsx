import React from "react";
import { render } from "@testing-library/react";
import preloadAll from "jest-next-dynamic";

import Home from "./page";

beforeAll(async () => {
  await preloadAll();
});

describe("test", () => {
  it("renders the page", async () => {
    const { container } = render(<Home />);

    expect(container).toMatchSnapshot();
  });
});
