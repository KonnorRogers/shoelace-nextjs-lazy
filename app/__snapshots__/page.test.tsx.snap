// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test renders the page 1`] = `
<div>
  <main>
    <p>
      Loading...
    </p>
  </main>
</div>
`;
